src/librustc/session/config.rs

```diff
@@ -109,6 +109,7 @@ impl OutputType {
             OutputType::Bitcode => "bc",
             OutputType::Assembly => "s",
             OutputType::LlvmAssembly => "ll",
+            OutputType::Object if cfg!(target_os = "windows") => "obj",
             OutputType::Object => "o",
```

Comment:
Does this not trigger a dead code warning for one of the two arms depending on `target_os`?